src/main/java/com/denizenscript/depenizen/bukkit/clientizen/ClientizenEventScriptEvent.java

```diff
@@ -0,0 +1,103 @@
+package com.denizenscript.depenizen.bukkit.clientizen;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.ObjectFetcher;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.depenizen.bukkit.clientizen.network.DataDeserializer;
+import org.bukkit.entity.Player;
+
+import java.util.Map;
+
+public class ClientizenEventScriptEvent extends ScriptEvent {
+
+    // <--[event]
+    // @Events
+    // clientizen event
+    //
+    // @Group Clientizen
+    //
+    // @Switch id:<id> to only process the event if the identifier received from the client matches the specified matcher.
+    //
+    // @Triggers When the server receives an event from a clientizen client, this requires config option 'Clientizen.process events' in the Depenizen config.
+    //
+    // @Context
+    // <context.id> returns an ElementTag of the event identifier received from the client.
+    // <context.data> returns the context MapTag received from the client.
+    // <context.(key)> returns the value of the input key in the context map, if available.
+    //
+    // @Player Always.
+    //
+    // @Example
+    // # Use to listen to clientizen events with the id 'my_id' and give the player that sent it a free diamond.
```

Comment:
This is a perfect example of a terrible script - infinite free diamonds for malicious players as easy as `/cex serverevent id:my_id`. This belongs on a Clientizen Guide page as a 'what not to do' example.

For the meta here, should, uh, do a narrate or something generic probably.

Reply:
Changed the example